checks/process.go

```diff
@@ -187,6 +188,23 @@ func formatCommand(fp *process.FilledProcess) *model.Command {
 	}
 }
 
+func formatIO(fp *process.FilledProcess, lastIO *process.IOCountersStat, before time.Time) *model.IOStat {
+	diff := time.Now().Unix() - before.Unix()
```

Comment:
Those are for the container-level, we also want at the process-level. Basically you can just attach the result of `formatIO` to that message type as you did with `CollectorProc`